students/NatalieRodriguez/Lesson01/break_me.py

```diff
@@ -0,0 +1,18 @@
+def NameError():
+     input_variable = input ("Enter your name: ")
+print ("your name is" + input_variable)
+"""The term input_variable is not defined, should throw a name error"""
+
+"""Type Error"""
+'dogs' + 3
+"""this is an unsupported operand type - cannot add a string and an integer"""
+
+
+"""Syntax Error"""
+def SyntaxError() #writing the code without a colon throws a Syntax Error
+
+
+"""Attribute Error"""
+anInt = 8
+anInt.append(4)
+
```

Comment:
everything here is supposed to be invalid 🙂 .. but as you are starting out with the assignment, couple notes on styling which we will check in assignments:
* function names should be all lower case with underscores (called snake_case) -> `def name_error`
* var names are also snake_case -> `an_int`
* `""" """` triple quotes strings are common in doc strings for a function or multi-line strings, for comments use `# this is a comment`
* there are a lot more tedious spacing rules etc that I won't get into 😆 
* highly recommended to use something like pycharm or run the code via linter (pep8, pyflakes etc)